pandas/_libs/index.pyx

```diff
@@ -52,7 +52,7 @@ cpdef get_value_at(ndarray arr, object loc, object tz=None):
 
 
 # Don't populate hash tables in monotonic indexes larger than this
-_SIZE_CUTOFF = 1000000
+_SIZE_CUTOFF = 1_000_000
```

Comment:
I would be OK with this in other places as well if you have a way to easily identify; more readable for sure